src/cljs/rems/actions/request_comment.cljs

```diff
@@ -0,0 +1,134 @@
+(ns rems.actions.request-comment
+  (:require [re-frame.core :as rf]
+            [rems.actions.action :refer [action-form-view button-wrapper]]
+            [rems.atoms :refer [textarea]]
+            [rems.autocomplete :as autocomplete]
+            [rems.text :refer [text]]
+            [rems.util :refer [fetch post!]]))
+
+(rf/reg-fx
+ ::fetch-potential-commenters
+ (fn [user]
+   (fetch (str "/api/applications/reviewers") ; TODO separate API for commenters
+          {:handler #(do (rf/dispatch [::set-potential-commenters %])
+                         (rf/dispatch [::set-selected-commenters #{}]))
+           :headers {"x-rems-user-id" (:eppn user)}})))
+
+(rf/reg-event-fx
+ ::open-form
+ (fn [{:keys [db]} [_]]
+   (prn (get-in db [:identity :roles]))
+   (merge {:db (assoc db
+                      ::comment ""
+                      ::potential-commenters #{}
+                      ::selected-commenters #{})}
+          (when (contains? (get-in db [:identity :roles]) :approver) ; TODO handler role?
+            {::fetch-potential-commenters (get-in db [:identity :user])}))))
+
+
+
+;; TODO together with application.cljs
+(defn enrich-user [user]
+  (assoc user :display (str (:name user) " (" (:email user) ")")))
+
+(rf/reg-event-db
+ ::set-potential-commenters
+ (fn [db [_ users]]
+   (assoc db ::potential-commenters (map enrich-user users))))
```

Comment:
`::potential-commenters` is initialized to be a set, but this makes it a seq.

Reply:
Some kind of old bug